### PR TITLE
Fix typo in help output

### PR DESCRIPTION
**Pull request by 1084565**

### CODE REVIEW

File: modules/cli/src/main/java/sap/prd/cmintegration/cli/Commands.java [hunk 1]
```diff
@@ -117,7 +117,7 @@ static BackendType getBackendType(CommandLine commandLine) {
             try {
                 return BackendType.valueOf(commandLine.getOptionValue(CMOptions.BACKEND_TYPE.getOpt()));
             } catch(IllegalArgumentException e) {
-                throw new RuntimeException("Cannot retrieve backend type.", e);
+                throw new RuntimeException("Can not retrieve backend type.", e);
```
Comment (rodibrin) [inline modules/cli/src/main/java/sap/prd/cmintegration/cli/Commands.java]: _These two spellings [cannot/can not] are largely interchangeable, but by far the most common is “cannot” and you should probably use it except when you want to be emphatic: “No, you can not wash the dog in the Maytag.”_

File: modules/cli/src/main/java/sap/prd/cmintegration/cli/Commands.java [hunk 2]
```diff
@@ -342,15 +342,15 @@ private static BackendType getBackendType(CommandLine commandLine, String[] args
         String b = commandLine.getOptionValue(CMOptions.BACKEND_TYPE.getOpt());
         if(StringUtils.isEmpty(b)) {
             printHelp();
-            throw new CMCommandLineException(format("Cannot retrieve backend type. Provide common option '-%s'. Values: %s.", CMOptions.BACKEND_TYPE.getOpt(), asList(BackendType.values())));
+            throw new CMCommandLineException(format("Can not retrieve backend type. Provide common option '-%s'. Values: %s.", CMOptions.BACKEND_TYPE.getOpt(), asList(BackendType.values())));
```
Comment (1084565) [inline modules/cli/src/main/java/sap/prd/cmintegration/cli/Commands.java]:
```suggestion
            throw new CMCommandLineException(format("Cannot retrieve backend type. Provide common option '-%s'. Values: %s.", CMOptions.BACKEND_TYPE.getOpt(), asList(BackendType.values())));
```

File: modules/cli/src/main/java/sap/prd/cmintegration/cli/Commands.java (hunk 1 above)
Comment (1084565) [inline modules/cli/src/main/java/sap/prd/cmintegration/cli/Commands.java]:
```suggestion
                throw new RuntimeException("Cannot retrieve backend type.", e);
```

Comment (1084565) [reply modules/cli/src/main/java/sap/prd/cmintegration/cli/Commands.java]: fixed, thanks

**Pull request merged**